src/pages/CreateSession.js

```diff
@@ -23,14 +25,17 @@ function CreateSession({ ...props }) {
         <Theme>
             <PageContainer>
                 <img id="logo" alt="LettuceEat logo" width="200" src="https://lh3.googleusercontent.com/proxy/wnjwUBl43KN-3GFcp3U-w_OCHSa2JkzeGS8ofOt0xwTM1m8EH7K1C7kMp6Bxd9WrBB1Ngom1cFhTPQ6A9EuUR5kvrA2bVaXLdULZhElkr2H-WGa7-5gAsCu40BnFWX81snu4QP6x496ebNA47eQAPRyXiRwwnVTe8Q" />
-                <h1>LettuceEat</h1>
-                <p>Easy bill splitting</p>
+                <Title>LettuceEat</Title>
+                <H2>Easy bill splitting</H2>
+                <br />
```

Comment:
tbh prob kinda sus to just use two <br /> tags please lemme know if there's a better option HAHA